components/demo/demo-snippet.js

```diff
@@ -66,14 +70,16 @@ class DemoSnippet extends LitElement {
 
 	render() {
 		const dirAttr = this._dir === 'rtl' ? 'rtl' : 'ltr';
+		const skeleton = this.hasSkeleton ? html`<d2l-switch text="Skeleton" ?on="${this._skeleton}" @change="${this._handleSkeletonChange}"></d2l-switch>` : null;
 		return html`
 			<div class="d2l-demo-snippet-demo-wrapper">
 				<div class="d2l-demo-snippet-demo" dir="${dirAttr}">
 					<slot name="_demo"></slot>
 					<slot></slot>
 				</div>
 				<div class="d2l-demo-snippet-settings">
-					<d2l-switch text="RTL" ?on="${dirAttr === 'rtl'}" @change="${this._handleDirChange}"></d2l-switch>
+					<d2l-switch text="RTL" ?on="${dirAttr === 'rtl'}" @change="${this._handleDirChange}"></d2l-switch><br>
+					${skeleton}
```

Comment:
This switch ends up looking like this in demo snippets that opt into skeletons:

![Screen Shot 2020-09-16 at 6 13 20 PM](https://user-images.githubusercontent.com/5491151/93397816-5b38bc00-f848-11ea-8d70-e244bc0ea6c5.png)